### ISSUE TITLE
Plate reader for 570-590nm at SAFS?

### ISSUE THREAD

**Issue by shellytrigg** [RobertsLab/resources#1034]
@kubu4 do you know of any plate readers in SAFS that can read 570-590nm? 

Previously, I used the plate reader in Genome Sciences with Emma (that could read at 575nm), but it would be easier logistically if there's one at SAFS. I know the Seeb's lab had one but I'm remembering it might not have been possible to read in this range because there may not have been the correct filter for that, but I'm not certain.

**Comment by kubu4** [comment]
The Seeb Lab plate reader (which we now have in FTR 228) can read at 572nm. It requires filter slide A to be in the machine. The manual should be next to the machine (as well as any unloaded filter slides/wheels) and describes how to swap filters (this might not be necessary, if filter slide A is already loaded).

**Comment by shellytrigg** [comment]
Great! @kubu4 can you confirm that we do indeed have filter slide A ?

**Comment by kubu4** [comment]
The machine has both filter slide A (which is supplied by Perkin Elmer) and filter slide B (which requires the addition of custom filters in the slide).

**Comment by kubu4** [comment]
Perking Elmer is the plate reader manufacturer, btw.

**Comment by shellytrigg** [comment]
@kubu4 great. To double check, can you confirm that filter slide A will work for the requirements of this calcium assay (see p. 5 and 11) for absorbance requirements. 
https://github.com/RobertsLab/resources/blob/master/protocols/Commercial_Protocols/Caymen_CalciumAssayKit_96well.pdf

**Comment by kubu4** [comment]
It should, since the plate reader (with filter slide A) can read at 572nm and the assay says it requires a plate reader that can measure absorbance from 570nm - 590nm.

**Comment by shellytrigg** [comment]
Great, thanks for confirming. That was my line of thinking. My concern was preparing the assay and then not being able to properly read the plate. I will head in Monday to set up the plate reader before i prepare the assay.

**Comment by kubu4** [comment]
Hmmm, after reading through the manual, combined with the fact that I'm not familiar enough with the software, I'm not sure if you can use the 572nm filter for a photometric assay (i.e. where absorbance is determined). The manual indicates that each filter is demarcated by a letter indicating its usage type (e.g. P450 is for photometric assays at 450nm, while D572 is for time-resolved fluorometry), suggesting that selecting a given filter is only possible in within the type of assay selected.

However, you can also create user-defined assay, but, again, it's not clear to me from the manual if the user can define a photometric assay using a filter designated for use in a different assay type (e.g. time-resolved fluorometry assay).

I think you'll need to poke around the software for a bit and see what you can figure out.

**Comment by shellytrigg** [comment]
I think I'll try to call tech support to ask about it. Thanks for pointing this out

**Comment by shellytrigg** [comment]
@kubu4 do you know the serial # by chance?

**Comment by shellytrigg** [comment]
I don't need the serial number anymore. I just got off the phone with tech support and they said I can create a user-defined protocol to only use the excitation filter 555nm to measure the absorbance. That is a filter that Chris Monson purchased and installed, which has a bandwidth of +/- 38nm and can accommodate the 570-590nm range I'm looking to read. I'll head in Tuesday to mess around with the software

**Comment by kubu4** [comment]
Cool. User-installed filters are in slide B. Not sure how they get labelled, though...

**Comment by shellytrigg** [comment]
I might ask Chris about it since he installed it. Also, if this doesn't end up working Emma said she can read my plates using the plate reader in GS

**Comment by kubu4** [comment]
> I might ask Chris about it since he installed it

Yeah, you're going to need to ask him. I looked at slide filter B this morning and there are two filters in the slide - neither is labelled. So, w/o talking to Chris, I'm not sure if there's a means of determining which wavelengths the two filters handle.

**Comment by shellytrigg** [comment]
Chris installed the 555nm filter in the filter wheel and set it up to read fluorescence only. To allow this filter to be used to read absorbance, I had to go into the filter menu listed on the main toolbar, open the settings for the 555nm filter and check off the absorbance box. When I did this and went to edit an absorbance protocol, I was then able to select the 555nm filter as the filter option. The protocol I created is called "Absorbance @ 570 (1.0s)"
